recipe/meta.yaml

```diff
@@ -36,3 +36,4 @@ extra:
   recipe-maintainers:
     - jakevdp
     - ocefpaf
+    - domoritz
```

Comment:
Thanks @domorits! Can you confirm that the home url moved to `https://github.com/vega/ipyvega`?
(If so we can update that info in this PR.)